api/audit/migrations/0005_auditlog_master_api_key.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.13 on 2022-05-31 07:39
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Couldn't we just delete this migration and recreate it to avoid the merge migration below?